transformer/src/main/scala/uk/ac/wellcome/transformer/modules/StreamRecordProcessorFactoryModule.scala

```diff
@@ -40,21 +39,11 @@ class StreamsRecordProcessor(receiver: RecordReceiver)
   }
 }
 
-class StreamsRecordProcessorFactory(
+class StreamsRecordProcessorFactory @Inject()(
```

Comment:
So if i'm understanding correctly it's appropriate to use the Module/Provider pattern when dependent on external config? https://twitter.github.io/finatra/user-guide/getting-started/modules.html